static/app/components/replays/replayTagsTableRow.tsx

```diff
@@ -0,0 +1,82 @@
+import React, {ReactElement} from 'react';
+import styled from '@emotion/styled';
+import {LocationDescriptor} from 'history';
+
+import {KeyValueTableRow} from 'sentry/components/keyValueTable';
+import Link from 'sentry/components/links/link';
+import Tooltip from 'sentry/components/tooltip';
+import Version from 'sentry/components/version';
+import {EventTag} from 'sentry/types/event';
+import {AnnotatedText} from 'sentry/components/events/meta/annotatedText';
+
+type Tag = {key: string; value: string[]};
+
+interface Props {
+  generateUrl: (tag: EventTag) => LocationDescriptor;
+  query: string;
+  tag: Tag;
+  meta?: Record<any, any>;
+}
+
+function ReplayTagsTableRow({tag, query, generateUrl, meta}: Props) {
+  const keyMetaData = meta?.key?.[''];
+  const valueMetaData = meta?.value?.[''];
+
+  const renderTagValue = () => {
+    switch (tag.key) {
+      case 'release': {
+        return tag.value.reduce((acc, value, index) => {
+          acc.push(<Version key={value} anchor={false} version={value} withPackage />);
+          if (index !== tag.value.length - 1) {
+            acc.push(', ');
+          }
+          return acc;
+        }, [] as (ReactElement | string)[]);
+      }
+      default: {
+        return tag.value.map((value, index) => {
```

Comment:
Oh didn't see this, but yeah can't we do something similar to this for `release` case?